src/components/pages/Home/Onboarding/Steps/ClaimTokens.tsx

```diff
@@ -0,0 +1,157 @@
+import { graphql, useStaticQuery } from 'gatsby'
+import axios from 'axios'
+import React, { ReactElement, useEffect, useState } from 'react'
+import { toast } from 'react-toastify'
+import { OnboardingStep } from '..'
+import { useWeb3 } from '../../../../../providers/Web3'
+import StepActions from '../../../../organisms/Onboarding/StepActions'
+import StepBody from '../../../../organisms/Onboarding/StepBody'
+import StepHeader from '../../../../organisms/Onboarding/StepHeader'
+import { GX_NETWORK_ID } from '../../../../../../chains.config'
+import { getErrorMessage } from '../../../../../utils/onboarding'
+
+const query = graphql`
+  query ClaimTokensQuery {
+    file(
+      relativePath: { eq: "pages/index/onboarding/steps/claimTokens.json" }
+    ) {
+      childStepsJson {
+        title
+        subtitle
+        body
+        image {
+          childImageSharp {
+            original {
+              src
+            }
+          }
+        }
+        gxButtonLabel
+        existingGxBalance
+        gxSuccess
+        oceanButtonLabel
+        existingOceanBalance
+        oceanSuccess
```

Comment:
issue(blocking): A very long query for this component. It seems like this can be simplified, e.g:
```jsx
const query = graphql`
  query ClaimTokensQuery {
    file(
      relativePath: { eq: "pages/index/onboarding/steps/claimTokens.json" }
    ) {
      childStepsJson {
        title
        subtitle
        //...
        buttons {
          name
          label
          balance
          success
       // ...
```

Enables adding additional buttons easier as a bonus